sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -598,4 +598,19 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 INSERT INTO categories(`id`,`name`, `value`, `parent`, `lft`, `rght`, `aco_spec`) select (select MAX(id) from categories) + 1, 'Invoices', '', 1, rght, rght + 1, 'encounters|coding' from categories where name = 'Categories';
 UPDATE categories SET rght = rght + 2 WHERE name = 'Categories';
 UPDATE categories_seq SET id = (select MAX(id) from categories);
+#EndIf
+
+#IfNotTable fee_schedule
+CREATE TABLE `fee_schedule` (
+    `id` BIGINT NOT NULL AUTO_INCREMENT,
+    `insurance_company_id` SMALLINT(4) DEFAULT NULL,
+    `plan` TINYTEXT(20),
```

Comment:
Any reason for not using varchar? TINYTEXT is still TEXT - with its engine overhead and limitations.

Reply:
Not untrue to an extent. Just ensure you set a length that is long enough However, don't dismiss TEXT where appropriate.

Reply:
For us TEXTs are for large objects. Occasionally it may be used to fudge the record length constraints but that still is a flag for specific db design constraint which forces admins to live with the overheads.

In this case such constraints are not obvious. So trying to learn from masters.

Reply:
again, not untrue. I just don't want Stephen to be fearful of using TEXT where needed. In this case you are entirely correct @mdsupport 
Concerning master, nope. Database design is an entire discipline on it's own and one i'd never claim to have mastered!:)